src/management/apps.ts

```diff
@@ -224,3 +234,64 @@ function getListAppsResourceString(projectId: string, platform: AppPlatform): st
 
   return `/v1beta1/projects/${projectId}${resourceSuffix}`;
 }
+
+/**
+ * Gets the configuration artifact associated with the specified a Firebase app
+ * @return a promise that resolves to configuration file content of the requested app
+ */
+export async function getAppConfig(
+  appId: string,
+  platform: AppPlatform
+): Promise<AppConfigurationData> {
+  try {
+    const response = await api.request("GET", getAppConfigResourceString(appId, platform), {
+      auth: true,
+      origin: api.firebaseApiOrigin,
+      timeout: TIMEOUT_MILLIS,
+    });
+    return parseConfigFromResponse(response.body, platform);
```

Comment:
This line should be outside of the `try` block.

Reply:
done